docs/number.adoc

```diff
@@ -2,28 +2,16 @@
 
 == Conversion between formatted decimals
 
-* `apoc.number.format(number) yield value` format a long or double using the default system pattern and language to produce a string
-* `apoc.number.format.pattern(number, pattern) yield value` format a long or double using a pattern and the default system language to produce a string
-* `apoc.number.format.lang(number, lang) yield value` format a long or double using the default system pattern pattern and a language to produce a string
-* `apoc.number.format.pattern.lang(number, pattern, lang) yield value` format a long or double using a pattern and a language to produce a string
+* `apoc.number.format(number)` format a long or double using the default system pattern and language to produce a string
 
-* `apoc.number.parseInt(text) yield value` parse a text using the default system pattern and language to produce a long
-* `apoc.number.parseInt.pattern(text, pattern) yield value` parse a text using a pattern and the default system language to produce a long
-* `apoc.number.parseInt.lang(text, lang) yield value` parse a text using the default system pattern and a language to produce a long
-* `apoc.number.parseInt.pattern.lang(text, pattern, lang) yield value` parse a text using a pattern and a language to produce a long
+* `apoc.number.parseInt(text)` parse a text using the default system pattern and language to produce a long
```

Comment:
these should have a short example like this, and a full with the optional parameters marked